src/Servers/IIS/Directory.Build.props

```diff
@@ -1,6 +1,13 @@
 <Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
+  
   <PropertyGroup>
     <KestrelSharedSourceRoot>$(MSBuildThisFileDirectory)..\Kestrel\shared\</KestrelSharedSourceRoot>
+    <HostArchDir>$(Platform)</HostArchDir>
+    <HostArchDir Condition="'$(Platform)' == 'Win32'">x86</HostArchDir>
+    <HostArchDir Condition="'$(Platform)' == 'AnyCPU'">x64</HostArchDir>
+    <LibNetHostPath>$(NetCoreTargetingPackRoot)\Microsoft.NETCore.App.Host.win-$(HostArchDir)\$(AppHostPackVersion)\runtimes\win-$(HostArchDir)\native</LibNetHostPath>
+    <!-- This tools version MUST match the same version as runtime uses to compile libnethost.lib -->
+    <PlatformToolsetVersion>v142</PlatformToolsetVersion>
```

Comment:
I'm surprised this works for users of VS2022 but saw the earlier comments about both toolsets being available in that release. Have you tested building w/ the native build grabbing `msbuild` from that release to be sure❔

Reply:
I only build with VS2022, so we are fine in that scenario, I don't think we will be able to support people who don't have VS2022 after this PR goes in, we probably will have to update our requirements to be the same as what runtime does here https://github.com/dotnet/runtime/blob/main/docs/workflow/requirements/windows-requirements.md#visual-studio

Reply:
We build from the command line on VS2019 in the CI. You haven't broken that.